lib/index-file.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 
-const Plugin = require('broccoli-plugin');
 const fs = require('fs');
 const path = require('path');
+const Plugin = require('broccoli-plugin');
 
-const BASE_MODULE = `
+function baseModule(buildTime) {
+  return `
   export const PROJECT_VERSION = '{{PROJECT_VERSION}}';
   export const PROJECT_REVISION = '{{PROJECT_REVISION}}';
```

Comment:
These should now also be added inline, because Rollup Replace is interpolating these too.